herbert_core/applications/multifit/@mfclass/simulate.m

```diff
@@ -144,7 +144,7 @@
 end
 
 % Mask the data
-[wmask, msk_out, ok, mess] = mask_data_for_fit (obj.w_, obj.msk_);
+[wmask, msk_out] = mask_data_for_fit (obj.w_, obj.msk_);
```

Comment:
Without the ok, mess returns on L.147, the next 3 lines won't work and have the potential to absorb a previously unset ok and/or an unset mess in the error_message.